net/openiked/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+name                openiked
+github.setup        openiked openiked-portable 7.0 v
+github.tarball_from archive
+categories          net
+platforms           darwin
+license             ISC
+maintainers         {@ryanakca debian.org:rak} \
+                    openmaintainer
+description         Internet Key Exchange (IKEv2) daemon
+long_description    A free implementation of the Internet Key Exchange (IKEv2) protocol \
+                    which performs mutual authentication and which establishes and maintains\
+                    IPsec VPN security policies and associations (SAs) between peers. It is \
+                    intended to be a lean, clean, secure, better configurable and \
+                    interoperable implementation that focusses on supporting the main standards \
+                    and most important features of IKEv2.
+homepage            https://www.openiked.org
+
+checksums           rmd160  e1a82859835e74d86fc01fca1c4148e4f39ec5e8 \
+                    sha256  4588d28471d20e491e616a44c71cf392e6cca8ea3af793a1f24c32d44101194c \
+                    size    302067
+
+depends_lib         port:libevent \
+                    path:lib/libssl.dylib:openssl
```

Comment:
There is now an `openssl` portgroup.  You can remove this dependency on the `openssl` port explicitly, and include the `openssl` portgroup instead:

```
PortGroup           openssl 1.0
```